ledger/lruboxes.go

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package ledger
+
+import (
+	"github.com/algorand/go-algorand/logging"
+)
+
+//msgp:ignore cachedResourceData
```

Comment:
@algoidurovic Sanity checking:  Is it by-design to refer to `cachedResourceData` here?  

If so, I'd appreciate a quick note to build my intuition.  Based on other cache implementations, I leave a _suggestion_.

```suggestion
//msgp:ignore cachedBoxData
```

Reply:
fixed.